### ISSUE TITLE
autoRehydrate() if and only if there is no internet connection

### ISSUE THREAD

**Issue by dharmawankenny** [rt2zz/redux-persist#396]
Hey, good library, been playing around with offline for a month now, ive tried redux-offline but i find that its not suitable to my use case, however, redux-persist did a great job fulfilling one of my needs (storing redux state easily and predictably),

However, during my time messing around with offline capability, i am wondering, is there a way for me to only autoRehydrate() when there is no internet connection, or possibly in lie-fie situation?

i know this is actually possible if i somehow wrap autoRehydrate() in some function but i am wondering is there an easier way, or is there someone who have encountered the same problem as me?

Basically, i need this library only for offline capability, i prefer to always fetch new data from the server when it is available and fall back to last stored data when offline (think like networkFirst strategy from service workers). Which is why i want to ignore autoRehydrating when i can fetch data from server.

Thanks in advance though, this library is great.

**Comment by dharmawankenny** [comment]
Closing this, just a simple if on configureStore when passing down enhancer should work.
